src/Mobs/Monster.h

```diff
@@ -217,6 +217,7 @@ class cMonster :
 
 	virtual void HandleFalling(void);
 	int m_LastGroundHeight;
+	int m_JumpCoolDown;
```

Comment:
No, use a regular `int` or `unsigned` for normal stuff. Architectures such as ARM prefer ints to bytes.